### ISSUE TITLE
Any implementation of depth-first search?

### ISSUE THREAD

**Issue by OndraSlejtr** [ProAlgos/ProAlgos-Cpp#80]
I see there is BFS implementation under review in issue #19, however only mention of DFS is in issue #25 that was closed by its author.
Can I give it a go?

**Comment by faheel** [comment]
As I mentioned in issue #70, I will be adding 4 classes for directed and undirected graphs using adjacency matrix and adjacency list representations, but only after issue #74 is taken care of.
Once the classes are added, you can use them to implement graph algorithms.

**Comment by faheel** [comment]
Issue #74 is now closed

**Comment by faheel** [comment]
Closing this since there haven't been any updates for a long time. Please open a new issue if you'd like to work on depth-first search.
